pkg/operation/seed/components.go

```diff
@@ -105,7 +105,7 @@ func defaultKubeScheduler(c client.Client, imageVector imagevector.ImageVector,
 	return gardenerkubescheduler.Bootstrap(c, secretsManager, v1beta1constants.GardenNamespace, image, kubernetesVersion)
 }
 
-func defaultGardenerSeedAdmissionController(c client.Client, imageVector imagevector.ImageVector, secretsManager secretsmanager.Interface) (component.DeployWaiter, error) {
+func defaultGardenerSeedAdmissionController(c client.Client, imageVector imagevector.ImageVector, secretsManager secretsmanager.Interface, kubernetesVersion *semver.Version) (component.DeployWaiter, error) {
```

Comment:
Sometimes it's called `seedVersion`, sometimes it's called `kubernetesVersion` in this file. Can you harmonize this?